app/uk/gov/hmrc/eoricommoncomponent/frontend/views/subscription/reg06_eori_already_linked.scala.html

```diff
@@ -30,12 +30,13 @@
 }
 @layout_di(messages("cds.reg06.outcome.eori-already-linked.title", longName), suppressTelephoneNumberDetection = false) {
     <div class="column-two-thirds">
-        <div class="govuk-box-highlight">
+        <div class="">
```

Comment:
Class can be totally removed here and you can have just `<div>`

Comment for master version later, not hotfix.

It's applying to all places where we have this